apps/cow-fi/app/(main)/legal/cowswap-privacy-policy/page.tsx

```
@@ -1,12 +1,15 @@
 'use client'
 
 import { Color } from '@cowprotocol/ui'
+import { initGtm } from '@cowprotocol/analytics'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)